app/services/build_claim_pdf_file_service.rb

```diff
@@ -0,0 +1,217 @@
+class BuildClaimPdfFileService # rubocop:disable Metrics/ClassLength
```

Comment:
This is a big class because the pdf is complex.  I resisted the urge to keep rubocop happy and break this down as I want to see the whole pdf generation details in one place - not spread across multiple files.

As you can see, the majority of the class is just saying 'write this value to this field'